### ISSUE TITLE
Noise on the first input channel of BlackHole 16c

### ISSUE THREAD

**Issue by giacecco** [ExistentialAudio/BlackHole#361]
I went back to BlackHole after some time without using it successfully, and realised that now I cannot use its first input because it outputs some kind of high-frequency distortion, as shown in the video [here](https://youtu.be/rSJbAiAEYEI). 

I am using BlackHole16ch.v0.2.9 on macOS 10.15.7, and BlackHole is the first device in an aggregate device so to exchange inputs and outputs between Bitwig Studio (using the aggregate) and VCV Rack (using BlackHole). Specifically, I am sending the CV for an arpeggio using a Bitwig's "HW CV Instrument" to a synth simulated in VCV rack. Anything from output 2 upwards works correctly, but output 1 doesn't: I hear some kind of noise instead. If I also send the Bitwig CV output to it, it seems like its voltage is "summed" to it.  

Any ideas? Thanks.

**Comment by devinroth** [comment]
Super strange. I’m not handling channel 1 any differently.

Try a few things and report back.

Does it happen when BlackHole isn’t the first in the aggregate?

How about BlackHole along without an aggregate? You’ll need to record audio to check.

Devin Roth

**Comment by giacecco** [comment]
Thanks for getting back to me so quickly @devinroth. Using BlackHole in a different position in the aggregate gives me the same problem I described. Using BlackHole on its own, I can't record any noise coming from channel 1. What can I try next?

**Comment by devinroth** [comment]
I wonder if it’s related to a couple other bugs reported.

Are you comfortable with Xcode?

Devin Roth

**Comment by giacecco** [comment]
Not really, but try pointing me to the instructions you've given to others. 

Also, where may I download an older version, say 0.2.7? 

I've also noticed something else that is very interesting. There is noise coming out of output 1 only if I send something to input 1.

![image](https://user-images.githubusercontent.com/303693/114311570-9d309480-9aef-11eb-96e0-9d39d4fc0935.png)

... but if I send nothing to input 1, there is nothing coming out of output 1: 

![image](https://user-images.githubusercontent.com/303693/114311606-bd605380-9aef-11eb-9839-e58a3c94613d.png)
